gobblin-metastore/src/test/java/org/apache/gobblin/metastore/testing/TestMetastoreDatabaseServer.java

```diff
@@ -152,10 +152,12 @@ private MySqlJdbcUrl getBaseJdbcUrl() throws URISyntaxException {
   }
 
   private MySqlJdbcUrl getInformationSchemaJdbcUrl() throws URISyntaxException {
+    // embedded mysql has an empty password by default
+    String password =  this.embeddedMysqlEnabled ? "" : ROOT_PASSWORD;
```

Comment:
I see EMBEDDED_MYSQL_ENABLED_FULL_KEY by default is set to be true, just curious when it's running in CI, how the config will be set as false?

Reply:
on CI it doesn't use embedded mysql, it talks to a MYSQL instance that's configured in the test environment.